tests/integration/test_api_gateway.py

```diff
@@ -127,3 +148,38 @@ def forward_request(self, **kwargs):
 
     # clean up
     proxy.stop()
+
+
+def test_api_gateway_lambda_proxy_integration():
+    test_port = 12123
+    backend_url = 'http://localhost:%s%s' % (test_port, API_PATH_LAMBDA_PROXY_BACKEND)
+
+    # create target HTTP backend
+    class TestListener(ProxyListener):
```

Comment:
Also, why do we actually need this backend service here? Isn't the purpose of this method to test the integration: client request -> API Gateway -> Lambda function ?

Having said that, in order to test this integration end-to-end, we will also need to create a Lambda function that will be invoked by API Gateway.